### PR TITLE
JIRA 5

### PR DESCRIPTION
**Pull request by ProgrammerFetch**
<sup> Created from JetBrains using [CodeStream](https://codestream.com/?utm_source=cs&utm_medium=pr&utm_campaign=github*com)</sup>

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1,6 +1,8 @@
 print("Here is some text!")
 print("and one more string")
 print("and one more string 2")
+print("and one more string 3")
```
Comment (BorovlevAS) [inline main.py:4]: test string 3

Comment (ProgrammerFetch) [reply main.py:4]: ok

File: main.py [hunk 2]
```diff
@@ -1,6 +1,9 @@
...
+print("and one more string 3")
```
Comment (BorovlevAS) [inline main.py:6]: text?

**Review by BorovlevAS — APPROVED** [review]
ok

**Pull request merged**